### DISCUSSION TITLE
Where to build this board

### DISCUSSION THREAD

**Discussion by watchever** [will127534/StarlightEye#7] [Q&A]
Hi Will, which supplier do you use to build your board. I have tried JPLPCB and upload the board on their website everything okay but sounds like ordering the IMX585 through JPLPCB is complicated. So if you have a good address I am interested.
Many thanks

**Comment by will127534** [comment]
What I do is that I have JLCPCB handle the component side, and I use low temperature paste for the CMOS sensor side.
I don't think JLCPCB has IMX585 nor can they solder it for you (I tried to send the datasheet to them to review the new parts).

**Comment by watchever** [comment]
Thanks Will, yes I see on JLCPCB that some components are missing. Not easy to find similar products in stock. Regarding the IMX585 sensor, it is also hard to find alone. Do you have any supplier to suggest? Thanks

**Reply by will127534** [reply]
Which component other then the CMOS sensor? I can suggest the replacement parts.

I only have some local supplier for the cmos sensor though, they are not on Alliexpress ...etc

**Comment by watchever** [comment]
Hi Will, when I upload the bom and position on JLCPCB, that's what I get. I guess that message about missing components. I think that most of them are pins, but some are electronic components and not sure which components I can use to replace. Some of them are shortfall but I think I can order them. Thanks for your help.
![Capture d'écran 2024-04-01 122841](https://github.com/will127534/StarlightEye/assets/14092607/4d6fbf2e-64ba-4eaa-818f-dbdb0868886a)

**Comment by watchever** [comment]
Hello Will, any feedback regarding my previous request. I would be really happy to test and use this open source sensor. It is really a great opensource project but if nobody can build it... That's a lost work which would be really too bad.  If you have any supplier to suggest it would be really nice. Thanks for your help.

**Reply by will127534** [reply]
I'm pretty sure you can order those parts with global sourcing.

On the sidenote, for OneInchEye project, people has successfully build their own boards so I'm not too worried that no one can build this, if you are not familiar with PCB design I highly recommend you go get the IMX585 board from [SOHO](https://soho-enterprise.com/wp/wp-content/uploads/2024/02/SE-SB8M-IMX585-Brochure202402.pdf).

**Reply by watchever** [reply]
Hi Will, is the new 5V power supply on the last version necessary if I don't need the IR Cut switch ?
Thanks
